### PR TITLE
Adding netbeans and netbeans-l10n as submodules

### PR DESCRIPTION
**Pull request by shivam71**
Added netbeans and l10n as  submodules and  made required changes to support them.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "netbeans-l10n"]
+	path = netbeans-l10n
+	url = https://github.com/apache/netbeans-l10n.git
+[submodule "netbeans"]
```
Comment (naren2605) [inline .gitmodules:4]: @shivam71  where can we mention specific commit id which checkout would be pointing to ?

File: netbeans
Comment (shivam71) [inline netbeans:1]: @naren2605 Here the commit hash is stored

File: .gitmodules (hunk 1 above)
Comment (shivam71) [reply .gitmodules:4]: The .gitmodules just stored the url not the commit hash . See the changed files in this commit it shows netbeans also which is a special file just storing the commit hash.

**Comment by naren2605** [comment]
LGTM

**Review by naren2605 — APPROVED** [review]

**Review by sid-srini — APPROVED** [review]
Thank you @shivam71 for improving the build experience and reducing human error in setup by getting this done. LGTM

**Pull request merged**